### ISSUE TITLE
Problem with migrating from EF6 to EF Core

### ISSUE THREAD

**Issue by tuscen** [dotnet/efcore#12269]
Hi. I'm migrating a project from EF6 to EF Core 2.1 and have a problem. I used scaffold to create a new context class. Almost everything was good however it didn't inlined existing primary key names. EF6 generated names for primary keys include their schema name (`PK_dbo_`). The problem is that EF Core algorithm for generating PK names don't include schema name so that after I create a new migration primary key names don't include schema name (`PK_`). This will affect subsequent migrations unless I either manually add names for every primary key or somehow influence the mechanism that generates names for primary keys. Is there some setting in EF Core so I can avoid this tedious task?

**Comment by tuscen** [comment]
After a bit of googling I've found a solution:

```c#
foreach (var entity in modelBuilder.Model.GetEntityTypes())
{
    foreach (var property in entity.GetProperties())
    {
        var primarykeyName = $"PK_{entity.Relational().Schema}.{entity.Relational().TableName.Pluralize()}";
        property.DeclaringEntityType.FindPrimaryKey().Relational().Name = primarykeyName;
    }
}
```

`.Pluralize()` is from `Humanizer` package.

Now primary key names in the generated migration is the same as in the database.

**Comment by bricelam** [comment]
Duplicate of #12821 this will be fixed in EF Core 2.2
